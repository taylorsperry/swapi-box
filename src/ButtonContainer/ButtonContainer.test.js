import React from 'react';
import { shallow } from 'enzyme';

describe('displayVehicles, fetchVehicles, cleanVehicles', () => {
  it('should redirect to /vehicles if there are vehicles in the store', () => {

  })

  it('should call fetchVehicles if there are no vehicles in the store', () => {

  })

  it('should call fetchAnything with the correct url', () => {

  })

  it('should call cleanVehicles with the correct data', () => {

  })

  it('should return an array of objects with the correct keys when cleanVehicles is called', () => {

  })

  it('should call storeVehicles when cleanVehicles is called', () => {
    
  })


})