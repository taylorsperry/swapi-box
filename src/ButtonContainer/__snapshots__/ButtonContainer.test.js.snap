// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonContainer matches the snapshot 1`] = `
<section
  className="btn-container"
>
  <NavLink
    className="button"
    id="people"
    name="people"
    onClick={[Function]}
    to="/people"
  >
    People
  </NavLink>
  <NavLink
    className="button"
    name="planets"
    onClick={[Function]}
    to="/planets"
  >
    Planets
  </NavLink>
  <NavLink
    className="button"
    name="vehicles"
    onClick={[Function]}
    to="/vehicles"
  >
    Vehicles
  </NavLink>
  <button
    className="button"
    name="Favorites"
    onClick={[Function]}
  >
     Favorites
  </button>
</section>
`;
